app/config/config_ezplatform_page.yml

```diff
@@ -29,6 +29,10 @@ parameters:
     eztags.default.tag_view.pagelayout: 'pagelayout.html.twig'
     # overwrites default tag related content limit
     eztags.default.tag_view.related_content_list.limit: 10
+    # community metrics location ID
+    community_metrics.location_id: 222
```

Comment:
@SylvainGuittard fixed in https://github.com/ezsystems/ezplatform-com/pull/113/commits/63d6c4ad89994f1a23aef57655a949fc7dbedf96